plugin.program.utorrent/resources/settings.xml

```diff
@@ -1,7 +1,9 @@
 <?xml version="1.0" encoding="UTF-8" standalone="yes"?>
 <settings>
 	<setting id="ip" type="text" label="30100" default="192.168.1.1" />
+	<setting id="use_https" type="bool" label="30105" default="false" />
 	<setting id="port" type="text" label="30101" default="8080" />
+	<setting id="path" type="text" label="30106" default="/gui/" />
```

Comment:
may be change the code to just set pathname here is better (default="gui"). otherwise user forget each "/" will fail to access.